cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/KafkaCruiseControlConfig.java

```diff
@@ -135,7 +135,7 @@
   /**
    * <code>broker.metrics.windows.ms</code>
```

Comment:
Nit: can we also update the relevant comment: `broker.metrics.windows.ms` -> `broker.metrics.window.ms`?